### ISSUE TITLE
Interception of incomplete commands

### ISSUE THREAD

**Issue by Sheridan** [zmeyc/telegram-bot-swift#38]
The bot engine intercepts incomplete commands. For example, if you pass a command `/g` to the hello-bot, it will react as if it were passed to `/greet`

**Comment by zmeyc** [comment]
This is intended behavior. For example instead of "zone list" user can type "z list".
They're also supported in subcommands, i.e. `router["zone list"] = f()` will also react to "z l".
Probably we need a flag to optionally disable abbreviations.

**Comment by Sheridan** [comment]
Yes, we need a flag, because many still remember IRC and write `/me ...`. If the bot has a command `/menu`, then it call :(
